build.sbt

```diff
@@ -24,6 +24,7 @@ val appName = "vat-obligations"
 lazy val appDependencies: Seq[ModuleID] = compile ++ test()
 lazy val plugins: Seq[Plugins] = Seq.empty
 lazy val playSettings: Seq[Setting[_]] = Seq.empty
+val bootstrapPlayVersion = "7.4.0"
```

Comment:
Why do we need to have a variable here, why can't we just specify the version number.